gems/pending/spec/appliance_console/database_maintenance_spec.rb

```diff
@@ -0,0 +1,99 @@
+require "appliance_console/prompts"
+require "appliance_console/database_maintenance"
+require "fileutils"
+require "tempfile"
+require "tmpdir"
+
+describe ApplianceConsole::DatabaseMaintenance do
+  DIRNAME = File.dirname(__FILE__).freeze
+
+  before do
+    allow(subject).to receive(:say)
+  end
+
+  describe "#ask_questions" do
+    before do
+      allow(subject).to receive(:say)
+      allow(subject).to receive(:clear_screen)
+      allow(subject).to receive(:agree)
+    end
+
+    context "when an hourly cron job does not exist" do
+      before do
+        @test_hourly_cron = "#{Dir.tmpdir}/miq-pg-maintenance-hourly.cron"
```

Comment:
This doesn't have to be an instance variable if it isn't being referenced in the specs.